experimental/oak_baremetal_app_crosvm/Cargo.toml

```diff
@@ -11,13 +11,13 @@ members = ["."]
 
 [features]
 default = ["vsock_channel"]
-vsock_channel = ["oak_baremetal_kernel/vsock_channel"]
-simple_io_channel = ["oak_baremetal_kernel/simple_io_channel"]
+vsock_channel = ["oak_restricted_kernel/vsock_channel"]
+simple_io_channel = ["oak_restricted_kernel/simple_io_channel"]
 
 [dependencies]
 log = "*"
 oak_baremetal_communication_channel = { path = "../oak_baremetal_channel" }
-oak_baremetal_kernel = { path = "../../experimental/oak_baremetal_kernel", default-features = false }
+oak_restricted_kernel = { path = "../../restricted_kernel", default-features = false }
```

Comment:
For consistency the crate name and folder name should probably the same (similar to #3275).

Reply:
Yes please

Reply:
I still think the `oak_` prefix is visual noise at the top level, but I've been outvoted :)